client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/support-nudge/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { recordMigrationInstructionsLinkClick } from '../tracking';
+import type { FC } from 'react';
+import './style.scss';
+
+type SupportNudgeProps = {
+	navigateToDoItForMe: () => void;
```

Comment:
I think it is better described when the event is triggered.

```suggestion
	onAskForHelp: () => void;
```

Reply:
Good call. Fixed in c4fd82921e6c1155c74fcb2dc7fc5ff13c1e9704.